03 Writing Algorithms/34 Algorithm Framework/02 Universe Selection/08 Options Universes/61 Option Chained Universe Selection.php

```diff
@@ -58,13 +58,13 @@
     return option_filter_universe.Strikes(-2, +2).FrontMonth().CallsOnly()</pre>
 </div>
 
-<?php
+<?
+$assetClass("Option");
```

Comment:
this is `$assetClass = "Option";` not a method

Reply:
yes already fixed